frontend/yarn.lock

```diff
@@ -11106,3 +11106,8 @@ yargs@^13.3.0:
     which-module "^2.0.0"
     y18n "^4.0.0"
     yargs-parser "^13.1.2"
+
+yarn@^1.22.4:
```

Comment:
Since we are switching completely to npm, can help to delete this file entirely?